tests/common/include/tests/common/material_interaction.inl

```diff
@@ -457,32 +461,37 @@ TEST(material_interaction, telescope_geometry_scattering_angle) {
 
         propagator_t::state state(bound_param, det);
 
-        state._stepping().set_overstep_tolerance(-1000.f * unit<scalar>::um);
+        // state._stepping().set_overstep_tolerance(-1000.f * unit<scalar>::um);
```

Comment:
E.g. should this be removed?

Reply:
Yeah I don't know why I put this one in the past - possibly because of the bug that is fixed now.
Technically, any value larger than zero should be fine because track is already on the surface